frameworks/core_foundation/views/view/enabled.js

```diff
@@ -204,6 +204,7 @@ SC.View.reopen(
     // Update the display if in a visible state.
     switch (state) {
     case SC.CoreView.ATTACHED_SHOWN:
+    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
```

Comment:
For me, this:

```
switch (state) {
    case SC.CoreView.ATTACHED_SHOWN:
    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
      // some code
    break;    
    case SC.CoreView.ATTACHED_BUILDING_IN:
      // some code
    break;
}
```

is not the same as:

```
switch (state) {
    case SC.CoreView.ATTACHED_SHOWN:
    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
      // some code
    case SC.CoreView.ATTACHED_BUILDING_IN:
      // some code
    break;
}
```

And I do agree that the second one really need a comment. But for the first one, I don't really see the need.